### ISSUE TITLE
Submenu is rendered in wrong place

### ISSUE THREAD

**Issue by Timmeeeey** [PerfectMemory/ngx-contextmenu#3]
**Describe the bug**
If you use html in an item template with submenu and click on the menu item text then the submenu is rendered in the wrong place.

Example:
``` html
<ng-template contextMenuItem [subMenu]="subMenu">
  <div>Item</div>
</ng-template>
```

**Stackblitz Example**
https://stackblitz.com/edit/angular-ivy-zcbqgj

![image](https://user-images.githubusercontent.com/46870283/168817018-69870d56-9391-47b1-84c9-8eaff06fb0ac.png)

**Comment by sroucheray** [comment]
Hi @Timmeeeey,

Thanks for the details. I think this might be because the menu has not enough space on the right to show itself, so it auto positions where it can and where it's best.

If I give it enough horizontal space on its right, I get this on your stackblitz :

On Firefox:
![image](https://user-images.githubusercontent.com/169818/168973808-44c91c90-d7e3-4d69-a78f-d49f1b02fcae.png)

On Chrome:
![image](https://user-images.githubusercontent.com/169818/168973908-5d47eac5-27c1-4ec4-a944-a7004685a038.png)

Could you check again and confirm please ?

**Comment by Timmeeeey** [comment]
Yes there is enough space. Did you click on the text "Item 2"?
![2022-05-18_08h59_14](https://user-images.githubusercontent.com/46870283/168977243-e4978256-985d-4b49-9d7c-4ea7e60dfde3.gif)

**Comment by sroucheray** [comment]
Fixed in 8.0.1. Thanks for the report @Timmeeeey ! https://stackblitz.com/edit/angular-ivy-njp6pu?file=angular.json
